src/components/App.tsx

```diff
@@ -16,14 +16,14 @@ import { KeyboardProvider } from '@src/context/Keyboard'
 import { ThemeContext } from '@src/context/Theme'
 import { addToHistory } from '@store/HistoryStore'
 import { dark, globals, light } from '@styles'
-import { formatInt, hasTrustedMint, isCashuToken, sleep } from '@util'
+import { formatInt, hasTrustedMint, isCashuToken, isErr, sleep } from '@util'
 import { initCrashReporting } from '@util/crashReporting'
 import { claimToken, isTokenSpendable, runRequestTokenLoop } from '@wallet'
```

Comment:
Why and where so you Call this?

Reply:
on line 159 in the catch block:

async function initDB() {
			try {
				await initDb()
				runRequestTokenLoop()
				// await addAllMintIds()
				/* const test = await getProofs()
				l({ test }, await getBalance())
				l(await getBalancesByKeysetId()) */
			} catch (e) {
				l(e)
				alert(`Something went wrong while initializing the DB! ${isErr(e) ? e.message : ''}`)
			}
		}